install/generator/04-syndesis-db.yml.mustache

```diff
@@ -227,4 +286,12 @@
           name: {{ Images.Support.Postgresql }}:{{ Tags.Postgresql }}
           namespace: ${POSTGRESQL_IMAGE_STREAM_NAMESPACE}
       type: ImageChange
-    - type: ConfigChange
+    - imageChangeParams:
+        automatic: true
+        containerNames:
+        - syndesis-db-metrics
+        from:
+          kind: ImageStreamTag
+          name: {{ Images.Support.PostgresExporter }}:{{ Tags.PostgresExporter }}
+          namespace: ${IMAGE_STREAM_NAMESPACE}
+      type: ImageChange
```

Comment:
File needs a new line at the end. When I ran the generator it produced an invalid YAML template with `type: ImageChange- apiVersion: v1` at line 773.